course/pages/dagster-essentials/extra-credit/4-coding-practice-grouping-assets.md

```diff
@@ -32,9 +32,13 @@ def name_of_asset():
 
 ### For the asset submodule method:
 
-For the `adhoc_request` asset, your code should look like this:
+For the `adhoc_request` asset, you can create a new file in the `defs` directory called `groups.py` and import the entire `requests` module to apply the `group_name`:
```

Comment:
The phrasing of, "import the entire `requests` module to apply the `group_name`" is confusing me a little bit.

Should we say import the assets in the requests module, and apply using the `load_assets_from_modules`?